install/devsds/local.conf

```diff
@@ -21,10 +21,13 @@
 # OpenSDS storage backend list, separated by a comma, support lvm right now.
 OPENSDS_BACKEND_LIST=lvm
```

Comment:
OPENSDS_BACKEND_LIST=lvm,nfs, don't have to define OPENSDS_BACKEND_LIST_NFS

Reply:
Please resolve erik's comments.